client/src/styles.scss

```diff
@@ -816,6 +820,10 @@ button.mat-menu-item.selected {
     font-size: 16px;
 }
 
+.overflow-hidden {
```

Comment:
That **could** cause whole vscroll lists or columns to flicker-to-disappear

Reply:
I removed overflow-hidden class and used the same classed as in agenda list.